system/mp/mpcomm.cpp

```diff
@@ -1120,8 +1153,10 @@ protected: friend class CMPPacketReader;
                 try {
                     s->shutdown();
                 }
-                catch (IException *) { 
+                catch (IException *e) {
                     socketfailed = true; // ignore if the socket has been closed
+                    WARNLOG("closeSocket() : Ignoring shutdown error");
```

Comment:
Perhaps put this in a #define _TRACELINKCLOSED so its associated with this tracing effort.